contents/blog/product-360.md

```diff
@@ -0,0 +1,126 @@
+---
+date: 2022-02-03
+title: I used to think you don't need product people. I was wrong.
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog", "Engineering"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/product-people.png
+featuredImageType: full
+---
+
+When PostHog was fresh out of YC last year, there were two things we focused on: (i) build and (ii) talk to customers. Hiring a product person felt like it would slow us down. Engineers on our team would speak to our users (also engineers) - surely they'd know what's best to prioritize? We didn't want our engineers to be "shielded" from our customers.
+
+By [PostHog's series B](/blog/15-million-series-b), our team showed me that I was totally wrong. Here's why that was the case, what we did about it, and the results.
+
+>James Hawkins is PostHog's Co-Founder & CEO. Follow him on [Twitter](https://www.linkedin.com/in/j-hawkins/).
+
+## Why I used to think this way
+
+My previous experience of product management:
+
+* Here's a list of features the enterprise sales team wants
+* How much pipeline does each of these features influence?
+* _Builds as much of the above as possible whilst trying to manage existing customer requests_
+
+Led to:
+
+* Lots of initial enterprise contracts
+* A product users find frustrating to use
+* No word of mouth growth
+* Customer churn in the long term
+
```

Comment:
A few tweaks to Neil's point to help make this flow a little better.

```suggestion
## Why I used to think this way

My previous experience of product management was something like this:

* Here's a list of features the enterprise sales team wants
* How much pipeline does each of these features influence?
* _Builds as much of the above as possible whilst trying to manage existing customer requests_

This lead to lots of initial enterprise contracts, but negative outcomes in the long term like:

* A product users find frustrating to use
* No word of mouth growth
* High customer churn

These were all traps I wanted to avoid for PostHog and lead me to favour hiring more engineering talent over product people.

```

Reply:
I think this is much better :)